### PR TITLE
chore(API-2025): upgrade legacy branch to Node 16

### PR DESCRIPTION
**Pull request by wjohnson-truework**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,8 @@
 dist
 .rpt2_cache/
 .rts2_cache_cjs/
+.rts2_cache_js/
 .rts2_cache_es/
 .rts2_cache_umd/
+node_modules
 .idea
```
Comment (wjohnson-truework) [inline .gitignore:8]:
```suggestion
node_modules
.idea

```

File: .github/workflows/cd.yml [hunk 2]
```diff
@@ -15,11 +15,10 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: 14
-          registry-url: "https://registry.npmjs.org"
```
Comment (wjohnson-truework) [inline .github/workflows/cd.yml:19]: Removed `registry-url` - this should not be needed

File: .github/workflows/cd.yml [hunk 3]
```diff
@@ -15,11 +15,10 @@ jobs:
...
+          node-version: 16
       - name: Installing dependencies
-        run: npm i
-      - name: Build and test
+        run: npm ci
```
Comment (wjohnson-truework) [inline .github/workflows/cd.yml:20]: `npm ci` instead of `npm i`

**Comment by wjohnson-truework** [comment]
If we think this is good, approve and will be glad to merge

**Review by christianbundy — APPROVED** [review]

**Pull request merged**